test/viewmodels/GeneralGuaranteeAccountViewModelSpec.scala

```diff
@@ -25,7 +25,9 @@ class GeneralGuaranteeAccountViewModelSpec extends SpecBase {
   "ViewModel" when {
 
     "account status is open" should {
-      val account = GuaranteeAccount("GAN1234", "EORI1234", AccountStatusOpen, Some(GeneralGuaranteeBalance(100, 100))) //scalastyle:off magic.number
+      val account = GuaranteeAccount("GAN1234", "EORI1234",
+        AccountStatusOpen, Some(GeneralGuaranteeBalance(100, 100))) //scalastyle:off magic.number
```

Comment:
Remove //scalastyle:off magic.number and create a meaningful constant for the magic numbers. Same at other places wherever  comment (//scalastyle:off magic.number) has been placed